packages/chaire-lib-backend/src/utils/processManagers/__tests__/TrRoutingProcessManager.test.ts

```diff
@@ -30,10 +31,12 @@ startProcessMock.mockImplementation(async (serviceName: string,
     service: tagName,
     name: serviceName
 }));
-
 beforeEach(function () {
     startProcessMock.mockClear();
     stopProcessMock.mockClear();
+    // Override max parallel setting
+    // TODO might be a better way to do this
```

Comment:
You could mock the config, which is sort of equivalent, though I guess mocking would be better since you are sure you are not touching the config for others. but I think jest runs each test separately so you shouldn't. But since you're asking...

Reply:
So I tried the mocking, but it did not work. Could not find an easy way to change the mock of the config in the test. If you have code example